pages/partials/questionScorePanel.ejs

```diff
@@ -10,8 +10,8 @@
       <% } %>
       <% if (assessment.type == 'Homework') { %>
       <tr>
-        <td>History:</td>
-        <td><%- include('../partials/questionAwardedPoints', {awarded_points: instance_question.variants_points_list, init_points: assessment_question.init_points});  %></td>
+        <td>Previous variants:</td>
+        <td><%- include('../partials/questionAwardedPoints', {awarded_points: instance_question.variants_points_list, init_points: assessment_question.init_points, instance_question_id: instance_question.id, previous_variants: instance_question_info.previous_variants});  %></td>
```

Comment:
This ends up being a really big vertical list. I'm wondering whether we should render this as a single cell using `colspan="2"` to let it use the full table width for wrapping?

Current style:
<img width="264" alt="image" src="https://user-images.githubusercontent.com/875833/232249440-432a2a2d-a99b-4b32-9426-749ca698298d.png">

Proposed change:
```
      <% if (assessment.type == 'Homework') { %>
      <tr>
        <td colspan="2" style="white-space: normal;">
          Previous variants:
          <%- include('../partials/questionAwardedPoints', {awarded_points: instance_question.variants_points_list, init_points: assessment_question.init_points, instance_question_id: instance_question.id, previous_variants: instance_question_info.previous_variants});  %>
        </td>
      </tr>
      <% } %>
```

With the proposed change:
<img width="263" alt="image" src="https://user-images.githubusercontent.com/875833/232249608-c483fbb6-0ad8-46db-944f-5228751640a6.png">

Reply:
Done in 94d9da247170bc5fee7b100a2a0cd39b3d11bcee